app/client/components/Clipboard.js

```diff
@@ -294,14 +294,27 @@ async function getTextFromClipboardItem(clipboardItem, type) {
 }
 
 /**
- * Helper to determine if the currently active element deserves to keep its own focus, and capture
- * copy-paste events. Besides inputs and textareas, any element can be marked to be a valid
- * copy-paste target by adding 'clipboard_focus' class to it.
+ * Helper to determine if the currently active element deserves to keep its own focus, and capture copy-paste events.
+ *
+ * By default, focus is automatically allowed if:
+ *   - there is no clipboard commands registered,
+ *   - the element is an input, textarea, select or iframe,
+ *   - the element has a tabindex attribute
+ *
+ * You can explicitly allow focus by setting different classes:
+ *   - using the 'clipboard_allow_focus' class will allow focusing the element having the class,
+ *   - using the 'clipboard_group_focus' class will allow focusing any descendant element of the one having the class
```

Comment:
This makes sense now, thank you. One potential small simplification: do you think it may be enough to only have `clipboard_allow_focus`, and to interpret it as allowing focus on the element OR any descendant (i.e. only keep `clipboard_group_focus` behavior, but I think `clipboard_allow_focus` would be a better name for it)? For the elements I can find in previous code that used the previous `clipboard_focus` class, I see no harm allowing focus on their descendants too.

Reply:
Hey there, thanks for the suggestion, I agree with you. I'd be willing to work on that in a later PR if that's ok with you.

I'd like to carefully verify if that assumption is correct, and at the same time I'd love to merge this PR soon. Changing this might lead to more work than we think right now.